back/src/Guide/Api/Methods.hs

```diff
@@ -163,6 +163,14 @@ deleteItem db requestDetails itemId = do
     addEdit db requestDetails edit)
   pure NoContent
 
+-- | Move item upper or lower
```

Comment:
"upper" and "lower" are not quite synonymical to "up" and "down".

You can use "up" and "down" everywhere, and for constructor names you can have `data Direction = DirectionUp | DirectionDown`.